pypeit/scripts/chk_edges.py

```diff
@@ -25,15 +25,20 @@ def get_parser(cls, width=None):
 
     @staticmethod
     def main(pargs):
-        from pypeit import edgetrace
+        from pypeit import edgetrace, slittrace
 
         # Load
         edges = edgetrace.EdgeTraceSet.from_file(pargs.trace_file, chk_version=(not pargs.try_old))
 
         if pargs.mpl:
             edges.show(thin=10, include_img=True, idlabel=True)
         else:
-            edges.show(thin=10, in_ginga=True)
+            # check if SlitTraceSet file exists(this allows to show the orders id instead of slitids)
+            slit_filename = slittrace.SlitTraceSet.construct_file_name(edges.traceimg.calib_key,
```

Comment:
What do you think of adding a parameter that allows the user to provide the slits file directly?  The Edges file and Slits file should be linked, but I could imagine a case where the user wants to specify the file directly (e.g., if it or the Edges file has been moved to a different directory).

Reply:
I have added it. Let me know what you think.

Reply:
I made a few direct changes.  Now, if the users file doesn't exist, it will still try to find the default file.

Reply:
Great! Thanks @kbwestfall!